crates/loans/src/lib.rs

```diff
@@ -2169,16 +2125,21 @@ impl<T: Config> LoansTrait<CurrencyId<T>, AccountIdOf<T>, BalanceOf<T>, Amount<T
         UnderlyingAssetId::<T>::try_get(lend_token_id).map_err(|_err| Error::<T>::InvalidLendTokenId.into())
     }
 
+    // NOTE: used in OracleApi, so don't use oracle calls here or it'll recurse forever
     fn recompute_collateral_amount(underlying: &Amount<T>) -> Result<Amount<T>, DispatchError> {
         // This function could be called externally to this pallet, with interest
         // possibly not having accrued for a few blocks. This would result in using an
         // outdated exchange rate. Call `accrue_interest` to avoid this.
         Self::ensure_active_market(underlying.currency())?;
         Self::accrue_interest(underlying.currency())?;
         let exchange_rate = Self::exchange_rate_stored(underlying.currency())?;
-        let underlying_amount = Self::calc_collateral_amount(underlying.amount(), exchange_rate)?;
+        let lend_amount = FixedU128::from_inner(underlying.amount())
```

Comment:
Minor nit: `lend_amount` seems less clear to me than `underlying_amount`

Reply:
Also same concern with `FixedU128` conversion (or lack thereof) probably not an issue but just want to make sure

Reply:
> Minor nit: `lend_amount` seems less clear to me than `underlying_amount`

Yea except that it's not the underlying amount :) it's returning lend tokens, not underlying tokens. It was misnamed previously